config/tests/guest/libvirt/migrate.cfg

```diff
@@ -0,0 +1,84 @@
+include tests-shared.cfg
+
+disk_target = sda
+setup_local_nfs = yes
+username = root
+password = 123456
+main_vm = avocado-vt-vm1-migration
```

Comment:
lets use same vm name across the configs..

Reply:
will change it